usaspending_api/disaster/v2/views/agency/loans.py

```diff
@@ -53,7 +53,7 @@ class LoansByAgencyViewSet(LoansPaginationMixin, ElasticsearchAccountDisasterBas
     required_filters = ["def_codes", "query"]
```

Comment:
Do the changes in this file actually make a difference to the aggregated results?